extensions/amp-ad/0.1/test/test-amp-ad-3p-impl.js

```diff
@@ -357,4 +357,165 @@ describes.realWin('amp-ad-3p-impl', {
       });
     });
   });
+
+  describe('full width responsive ad', () => {
+
+    const VIEWPORT_WIDTH = 300;
+    const VIEWPORT_HEIGHT = 600;
+
+    beforeEach(() => {
+      adConfig['_ping_'].fullWidthHeightRatio = 1.2;
+      win.document.body.removeChild(ad3p.element);
+    });
+
+    describe('should resize correctly', () => {
+      let impl;
+      let element;
+      let attemptChangeSizeSpy;
+
+      function constructImpl(config) {
+
+        config.type = '_ping_';
+        element = createElementWithAttributes(win.document, 'amp-ad', config);
+        document.body.appendChild(element);
```

Comment:
This might be the reason why the tests are failing. You're accessing global object which will affect all other tests.